gui/default/syncthing/core/syncthingController.js

```diff
@@ -780,6 +780,60 @@ angular.module('syncthing.core')
             return 'info';
         };
 
+        $scope.syncthingStatus = function () {
+            
+            var deviceCount = $scope.devices.length;
+            var syncCount = 0;
+            var notifyCount = 0;
+            var pauseCount = 0;
+            
+            //loop through all devices
+            for (var i = 0; i < $scope.devices.length; i++) {
+                var status = $scope.deviceStatus({
+                    deviceID:$scope.devices[i].deviceID
+                });
+                switch (status) {
+                    case 'syncing': syncCount++; break;
+                    case 'unknown': notifyCount++; break;
+                    case 'paused': pauseCount++; break;
+                    case 'unused': deviceCount--; break;
+                }
+            }
+
+            //loop through all folders
+            var folderListCache = $scope.folderList();
+            for (var i = 0; i < folderListCache.length; i++) {
+                var status = $scope.folderStatus(folderListCache[i]);
+                switch (status) {
+                    case 'syncing':
+                    case 'scanning':     //possible?
+                        syncCount++;
+                        break;
+                    case 'stopped':
+                    case 'unknown':
+                    case 'outofsync':
+                    case 'error':        //possible?
+                        notifyCount++;
+                        break;
+                }
+            }
+
+            //notifications
+            if ($scope.openNoAuth)                               notifyCount++;
```

Comment:
im not sure what you are referring to, but as far as i understand, the variables i set are all inside the `$scope.syncthingStatus = function () {}` block, so are local?